demo/readFormFields.js

```diff
@@ -1,150 +1,97 @@
 function GetSelectorForElement(el) {
-  const queryParts = [];
-  let curElement = el;
-  
-  while (curElement && curElement.parentNode) {
-    if (curElement.id) {
-      queryParts.unshift(`#${curElement.id}`);
-      break;
-    } else {
-      if (curElement.ownerDocument && (curElement === curElement.ownerDocument.documentElement || curElement === curElement.ownerDocument.body)) {
-        queryParts.unshift(curElement.tagName.toLowerCase());
-      } else {
-        let tagName = curElement.tagName.toLowerCase();
-        const siblings = curElement.parentNode.children;
-        let i = 0;
-        for (; i < siblings.length; i++) {
-          if (curElement === siblings[i]) {
+    var queryParts = [];
+    var curElement = el;
+    while (curElement && curElement.parentNode) {
+        if (curElement.id) {
+            queryParts.unshift("#" + curElement.id);
             break;
-          }
         }
-        queryParts.unshift(`${tagName}:nth-child(${i + 1})`);
-      }
-      curElement = curElement.parentElement;
+        else {
+            if (curElement.ownerDocument && (curElement === curElement.ownerDocument.documentElement || curElement === curElement.ownerDocument.body)) {
+                queryParts.unshift(curElement.tagName.toLowerCase());
+            }
+            else {
+                var tagName = curElement.tagName.toLowerCase();
+                var siblings = curElement.parentNode.children;
+                var i = 0;
+                for (; i < siblings.length; i++) {
+                    if (curElement === siblings[i]) {
+                        break;
+                    }
+                }
+                queryParts.unshift(tagName + ":nth-child(" + (i + 1) + ")");
+            }
+            curElement = curElement.parentElement;
+        }
+    }
+    return queryParts.join(' > ');
+}
+function ReadCookies() {
+    var cookies = document.cookie;
+    var coolist = cookies.split(' ');
+    if (document.getElementById('ins_cookies').innerHTML !== '') {
+        document.getElementById('ins_cookies').innerHTML = '';
+    }
+    else {
+        for (var _i = 0, coolist_1 = coolist; _i < coolist_1.length; _i++) {
+            var cookie = coolist_1[_i];
+            var content = cookie.split('=');
+            var li = document.createElement('li');
```

Comment:
it seems the extension code is creeping in here. This will make extension on its own useless soon